src/components/MultiReportTable.js

```diff
@@ -51,7 +52,12 @@ const Results = ({
                     .map((data, i) => (
                         <div key={i} className={setPageBreaks(i, pageBreaksAfter)}>
                             <div>
-                                <h3>{data.title.displayString}</h3>
+                                {formatTitle(
```

Comment:
Maybe slightly outside the scope of this pr but it looks like some of these divs could be cut? I think its maybe better to use react fragments instead of divs if they're being used to wrap multiple components.

Reply:
Yes. I was being lazy